tests/unit/services/extractVehicleBooking.test.ts

```diff
@@ -0,0 +1,106 @@
+import { extractVehicleBookings } from '../../../src/services/extractVehicleBooking';
+import logger from '../../../src/util/logger';
+import { GetDynamoStream } from '../resources/mTestResultRecords';
+
+jest.mock('../../../src/util/logger');
+
+describe('extractVehicleBooking', () => {
+  it('should return array of valid vehicle bookings for psv/hgv', () => {
+    const event = GetDynamoStream(1);
+
+    const result = extractVehicleBookings(event);
+
+    expect(result).toHaveLength(2);
+    expect(result[0]).toEqual({
+      name:'Rowe, Wuns',
+      bookingDate:'2021-01-14T10:36:33.987Z',
```

Comment:
It looks like the message format was missing from the Jira ticket.
- bookingDate and testDate need to be just date parts i.e. '2021-01-14'.
- testCode needs to be uppercase.